linera-service/src/server.rs

```diff
@@ -461,10 +495,12 @@ async fn run(options: ServerOptions) {
             max_stream_queries,
             cache_size,
         } => {
+            linera_version::VERSION_INFO.log();
+
             let genesis_config: GenesisConfig =
-                util::read_json(&genesis_config_path).expect("Fail to read initial chain config");
```

Comment:
Strictly, these `expect`s are incorrect anyway: the string given to `expect` should be a justification for why you think the thing _won't_ fail.

Reply:
I agree but we seem to have made an exception in CLI-related panics so far.